src/Microsoft.AspNet.Mvc.Razor/RazorViewEngineOptions.cs

```diff
@@ -42,5 +45,26 @@ public IFileProvider FileProvider
                 _fileProvider = value;
             }
         }
+
+        /// <summary>
+        /// Gets or sets the configuration name used by <see cref="RazorViewEngine"/> to compile razor views.
+        /// </summary>
+        /// <remarks>
+        /// At startup, this is initialized to "Debug" if <see cref="Microsoft.AspNet.Hosting.IHostingEnvironment"/> service is
+        /// registred and environment is development (<see cref="HostingEnvironmentExtensions.IsDevelopment"/>) else it is set to
+        /// "Release".
+        /// </remarks>
+        public string Configuration
+        {
+            get { return _configuration; }
+            set
+            {
+                if (value == null)
```

Comment:
Make this `string.IsNullOrEmpty` Generally use that to null checks for string values.